.github/workflows/ci.yml

```diff
@@ -81,21 +81,21 @@ jobs:
         os: [k8s-linux-runner]
     steps:
     - uses: actions/checkout@v3
-    - uses: actions-rs/toolchain@v1
+    - uses: radixdlt/public-iac-resuable-artifacts/rust-setup@feature/rust-coverage
```

Comment:
reusable?

Reply:
yeah there's a typo in the repo error. nothing we can do I'm afraid